gradle.properties

```diff
@@ -4,22 +4,22 @@ group=org.ballerinalang
 version=1.5.1-SNAPSHOT
 
 ballerinaLangVersion=2201.5.0
```

Comment:
```suggestion
ballerinaLangVersion=2201.6.0-20230519-135600-d887142d
```

Reply:
Updated

Reply:
@kalaiyarasiganeshalingam do we need to update the lang version?

Reply:
Sorry, we don't need to update to update6. I thought the HTTP lang version was updated to update 6.